btb/basic_btb/return_stack.cc

```diff
@@ -26,7 +26,7 @@ void return_stack::calibrate_call_size(champsim::address branch_target)
     auto call_ip = stack.back();
     stack.pop_back();
 
-    auto estimated_call_instr_size = champsim::offset(call_ip, branch_target);
+    auto estimated_call_instr_size = call_ip > branch_target ?  champsim::uoffset(branch_target, call_ip) : champsim::uoffset(call_ip, branch_target);
```

Comment:
Looks fine to me. It's definitely a good idea to limit the # of printouts since we don't know how common this is. I can test this later to see if it works well.

Reply:
Seems to work. Allows for the traces to run, but prints out the warning a max of 10 times.